crates/oxc_ast/src/ast/js.rs

```diff
@@ -1725,6 +1725,15 @@ pub struct ArrayPattern<'a> {
     pub rest: Option<Box<'a, BindingRestElement<'a>>>,
 }
 
+/// A `...rest` binding in an [array](ArrayPattern) or [object](ObjectPattern) destructure.
+///
+/// ## Examples
+/// ```ts
+/// const [a, ...rest] = [1, 2, 3];
+/// //           ^^^^  argument
+/// const { x, y, ...others} = foo.bar();
+/// //               ^^^^^^  argument
```

Comment:
```suggestion
/// const [a, ...rest] = [1, 2, 3];
/// //        ^^^^^^^  argument
/// const { x, y, ...others} = foo.bar();
/// //            ^^^^^^^^^  argument
```
nit: i think technically `BindingRestElement` includes the span for the `...`.  If you want to get the span for the argument, you have to use `GetSpan` trait and `.argument.span()`

Reply:
You're right, `BindnigRestElement` does. But these arrows are for `argument`, so they're correct